core/storage/user/gae_models.py

```diff
@@ -1599,6 +1682,12 @@ def get_deletion_policy():
         """
         return base_models.DELETION_POLICY.DELETE
 
+    @staticmethod
+    def get_export_policy():
+        """Model contains user data."""
```

Comment:
Docstring does not match what is returned. I think the rationale here is that this is a computed model and the information already exists in other models which are exported.

Reply:
Updated docstring.